include/core/boundary_conditions.h

```diff
@@ -357,7 +358,12 @@ namespace BoundaryConditions
                       Patterns::Double(),
                       "Value (Double) for constant temperature at bc");
 
-    prm.declare_entry("Tenv",
+    prm.declare_entry("h",
+                      "0",
+                      Patterns::Double(),
+                      "Value (Double) for the h coefficient of convection bc");
+
+    prm.declare_entry("Tinf",
```

Comment:
I feel that "Tinf" is less user friendly (as far as the .prm file is concerned) that "Tenv"

Reply:
I prefer Tinf, more what I am used to :). We can document it clearly in the wiki though.